### PR TITLE
docs: cli reference

### PR DESCRIPTION
**Pull request by ljcolling**
added cli arguments to docs

### CODE REVIEW

**Comment by tlestang** [comment]
Thanks, don't you need to `pip install cats` in the [workflow](https://github.com/GreenScheduler/cats/blob/main/.github/workflows/docs-build-deploy.yml)? In case autodocs needs to `import cats.__init__`. I can see that's one of the reason the API reference isn't being generated by sphinx (plus the fact that module names are outdated after merging #49)

**Comment by ljcolling** [comment]
> Thanks, don't you need to `pip install cats`

Ah yes you do. I'll update it

**Comment by ljcolling** [comment]
Ok, I think it should work as expected now

**Comment by tlestang** [comment]
Yes, I pushed your changes to my fork and built it from there: https://tlestang.github.io/cats/api-reference.html

File: docs/source/api-reference.rst [hunk 1]
```diff
@@ -47,25 +47,17 @@ Modules
...
+``cats.optimise_starttime``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. automodule:: cats.timeseries_conversion
+.. automodule:: cats.optimise_starttime
     :members:
 
-
 Python objects
 --------------
```
Comment (tlestang) [inline docs/source/api-reference.rst:60]: Do we need [this section](https://tlestang.github.io/cats/api-reference.html#python-objects)? It seems redundant with the module documentation above.

Comment (sadielbartholomew) [reply docs/source/api-reference.rst:60]: Good point Thibault, we can get rid of it. Can you remove the section here in this PR, please Lincoln?

**Review by sadielbartholomew — APPROVED** [review]
Confirmed to fix the issue at hand. Excellent, thanks Lincoln.

**Pull request merged**